alpine/Dockerfile

```diff
@@ -6,39 +6,35 @@ ARG SIA_PACKAGE="Sia-v${SIA_VERSION}-linux-amd64"
 ARG SIA_ZIP="${SIA_PACKAGE}.zip"
 ARG SIA_RELEASE="https://sia.tech/releases/${SIA_ZIP}"
 
-RUN apt-get update && apt-get install -y wget unzip
-
-RUN wget "$SIA_RELEASE" && \
-      mkdir /sia && \
-      unzip -j "$SIA_ZIP" "${SIA_PACKAGE}/siac" -d /sia && \
-      unzip -j "$SIA_ZIP" "${SIA_PACKAGE}/siad" -d /sia
+RUN apt-get update && \
+    apt-get install -y wget unzip && \
+    wget "$SIA_RELEASE" && \
+    mkdir /sia && \
+    unzip -j "$SIA_ZIP" "${SIA_PACKAGE}/siac" -d /sia && \
+    unzip -j "$SIA_ZIP" "${SIA_PACKAGE}/siad" -d /sia
 
 FROM alpine:3
 LABEL maintainer="NebulousLabs <devs@nebulous.tech>"
 LABEL autoheal=true
-ARG SIA_DIR="/sia"
-ARG SIA_DATA_DIR="/sia-data"
 
 RUN mkdir /lib64 && \
-      ln -s /lib/libc.musl-x86_64.so.1 /lib64/ld-linux-x86-64.so.2 && \
-      apk add --no-cache socat
-COPY --from=zip_downloader /sia/siac "${SIA_DIR}/siac"
-COPY --from=zip_downloader /sia/siad "${SIA_DIR}/siad"
+    ln -s /lib/libc.musl-x86_64.so.1 /lib64/ld-linux-x86-64.so.2 && \
+    apk add --no-cache socat
 
-# Workaround for backwards compatibility with old images, which hardcoded the
-# Sia data directory as /mnt/sia. Creates a symbolic link so that any previous
-# path references stored in the Sia host config still work.
-RUN ln -s "$SIA_DATA_DIR" /mnt/sia
```

Comment:
Is this no longer needed?

Reply:
I'm not entirely sure. Maybe we should keep it. We don't have a good reason to remove it, aside from wanting to clean up.